frontend/packages/kubevirt-plugin/src/utils/strings.ts

```diff
@@ -18,6 +18,38 @@ export const ADD_NETWORK_INTERFACE = 'Add Network Interface';
 export const ADD_SNAPSHOT = 'Take Snapshot';
 export const RESTORE_SNAPSHOT = 'Restore Snapshot';
 
+// provision sources descriptions
+export const PROVISION_SOURCE_PXE_DESC =
+  'Boot an operating system from a server on a network. This will show up as a NIC on the Networking step. Requires a PXE bootable network attachment definition';
+export const PROVISION_SOURCE_CONTAINER_DESC =
+  'Link to a bootable operating system container located in a registry accessible from the cluster. Example: kubevirt/cirros-registry-disk-demo. This action will not create a persistent volume claim (PVC). This will show up as a disk in the Storage step';
```

Comment:
Do we have to specify this will not create a PVC? Kubernetes use should probably know what container is.